examples/dao/assets/dao-fund-lsig.py

```diff
@@ -0,0 +1,54 @@
+import sys
+sys.path.insert(0,'..')
+
+from algobpy.parse import parse_params
+from pyteal import *
+
+def proposal_lsig(ARG_DAO_APP_ID):
+    """
+    Represents DAO treasury (ALGO/ASA)
+    """
+
+    # check no rekeying, close remainder to, asset close to for a txn
+    def basic_checks(txn: Txn): return And(
+        txn.rekey_to() == Global.zero_address(),
+        txn.close_remainder_to() == Global.zero_address(),
+        txn.asset_close_to() == Global.zero_address()
+    )
+
+    # ideally we should restrict the sender here (no one can withdraw funds)
```

Comment:
currently anyone with this DAO address can use it's funds, shouldn't we block it ?

Reply:
yes, we are restricting it. The comment should be updated.

Reply:
done. [https://github.com/scale-it/algo-builder/pull/458#discussion_r710951355](https://github.com/scale-it/algo-builder/pull/458#discussion_r710951355)